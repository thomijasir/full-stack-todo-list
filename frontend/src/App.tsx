import React from 'react';
import { BrowserRouter, Routes, Route, Navigate } from 'react-router-dom';
import LoginPage from './container/LoginPage/Login.page';
import RegisterPage from './container/RegisterPage/Register.page';
import Dashboard from './container/DashboardPage/Dashboard.page';
import './App.scss';

const App = () => {
  return (
    <BrowserRouter>
      <Routes>
        <Route path="/" element={<LoginPage />} />
        <Route path="/register" element={<RegisterPage />} />
        <Route path="/dashboard" element={<Dashboard />} />
        <Route path="*" element={<Navigate to={'/'} />} />
      </Routes>
    </BrowserRouter>
  );
};

export default App;
