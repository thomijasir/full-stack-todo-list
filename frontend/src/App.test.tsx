import React from 'react';
import renderer from 'react-test-renderer';
import App from './App';

test('Render Login Page', () => {
  const element = renderer.create(<App />).toJSON();
  expect(element).toMatchSnapshot();
});
