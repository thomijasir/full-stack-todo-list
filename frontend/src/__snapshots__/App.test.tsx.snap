// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Login Page 1`] = `
<div>
  <h1>
    Its Login Screen
  </h1>
</div>
`;
